samples/notebooks/Lightweight Python components - basics.ipynb

```diff
@@ -96,14 +96,14 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 5,
+   "execution_count": null,
    "metadata": {},
    "outputs": [],
    "source": [
     "#Advanced function\n",
     "#Demonstrates imports, helper functions and multiple outputs\n",
     "from typing import NamedTuple\n",
-    "def my_divmod(dividend: float, divisor:float) -> NamedTuple('MyDivmodOutput', [('quotient', float), ('remainder', float)]):\n",
+    "def my_divmod(dividend: float, divisor:float, output_dir:str = './') -> NamedTuple('MyDivmodOutput', [('quotient', float), ('remainder', float)]):\n",
```

Comment:
This change goes against the idea that outputs should be outputs.
The metrics and metadata files should have just been added to the outputs list.